packages/types/src/definitions/Clients.type.ts

```diff
@@ -0,0 +1,15 @@
+export type CustomFieldLabel = string;
+
+export enum ClientTypes {
+  Unknown = 'unknown',
+  Ontime = 'ontime',
+}
+
+export type Client = {
+  name: string;
+  type: ClientTypes | string;
```

Comment:
question? why the string? dont we have control over this?

Reply:
yes that was my thought
and I have the enum to make it easier to set in the server, but maybe that is bad?

Reply:
It is unclear what a best way would be, but I suggest keeping this simple for now.

how about a union like
```ts
export type ClientType = 'ontime' | 'unknown' | string;
```

I believe we dont get benefits from the enum since this is dynamic


Also, mega nitpick: the ClientType should not be plural
https://stackoverflow.com/questions/15755955/naming-of-enums-in-java-singular-or-plural